### PR TITLE
Handle messaging service disconnect error

### PR DESCRIPTION
**Pull request by fjvallarino**
@Rise-Vision/apps please review

### CODE REVIEW

File: web/scripts/displays/services/svc-primus.js [hunk 1]
```diff
@@ -68,16 +74,19 @@ angular.module('risevision.displays.services')
           });
         });
 
-      return deferred.promise.then(checkNewMSPresence.bind(null, displayIds));
+      return deferred.promise
+        .then(function (oldMSResults) {
+          return factory.checkNewMSPresence(displayIds, oldMSResults);
+        })
+        .catch(function () {
+          return factory.checkNewMSPresence(displayIds, []);
```
Comment (alex-deaconu) [inline web/scripts/displays/services/svc-primus.js]: @fjvallarino I think we should log an error here, at least in the console, to indicate the connection to Old MS failed. Thanks.

File: web/scripts/displays/services/svc-primus.js [hunk 2]
```diff
@@ -35,11 +35,17 @@ angular.module('risevision.displays.services')
     };
   }])
 
-  .factory('getDisplayStatus', ['loadOldPrimus', '$q', '$timeout', 'checkNewMSPresence', function (
-    loadOldPrimus, $q, $timeout, checkNewMSPresence) {
-    return function (displayIds) {
+  .factory('displayStatusFactory', ['loadOldPrimus', '$q', '$http', '$timeout', 'processErrorCode', 'MESSAGING_PRESENCE_URL',
```
Comment (alex-deaconu) [inline web/scripts/displays/services/svc-primus.js]: @fjvallarino I really thing we should split out this and screenshotRequester into separate files. I'm always confused when trying to find this service. Thoughts? Thanks.

**Comment by alex-deaconu** [comment]
@fjvallarino PR looks good to me 👍 . 2 suggestions for improvement. I'm assuming test failures aren't related to these code changes... Thanks.

**Comment by fjvallarino** [comment]
@alex-deaconu updated!

**Comment by alex-deaconu** [comment]
@fjvallarino I just tried to block `https://services-stage.risevision.com/messaging/presence` in Chrome, and the error message appeared correctly. However, in the console there was still the error for `Error checking presence on the legacy messaging service`. I think your catch statement catches both errors and logs them, instead of just old primus. Can you double check? Thanks.

**Comment by alex-deaconu** [comment]
@fjvallarino looks good, thanks for fixing. As mentioned, I had to re-run the staging step as it failed due to `apt-get` again. Thanks.

**Pull request merged**